admin/python/lsst/qserv/admin/dataLoader.py

```diff
@@ -684,17 +671,33 @@ def _updateCss(self, database, table):
         """
 
         # create database in CSS if not there yet
-        if not self.css.dbExists(database):
+        if not self.css.containsDb(database):
             self._log.info('Creating database CSS info')
             options = self.partOptions.cssDbOptions()
-            self.css.createDb(database, options)
+            striping = css.StripingParams(options['nStripes'], options['nSubStripes'],
+                                          0, options['overlap'])
+            self.css.createDb(database, striping, options['storageClass'], 'RELEASED')
 
         # define options for table
         options = self.partOptions.cssTableOptions()
-        options['schema'] = self._schemaForCSS(database, table)
-
-        self._log.info('Creating table CSS info')
-        self.css.createTable(database, table, options)
+        schema = self._schemaForCSS(database, table)
+
+        if options.get('match', False):
+            matchParams = css.MatchTableParams(options['dirTable1'], options['dirColName1'],
+                                               options['dirTable2'], options['dirColName2'],
+                                               options['flagColName'])
+            chunkLevel = 2 if options['subChunks'] else 1
```

Comment:
Same comment regarding match tables and sub-chunking as in qserv-admin.py. This whole code block also looks very similar to the code I commented on there - not sure it's worth it, but maybe factor into a helper function?